helm/values.yaml

```diff
@@ -67,6 +67,26 @@ serviceMonitor:
   # -- ServiceMonitor scrape timeout
   # scrapeTimeout: 10s
 
+## extra VolumeMounts for the pod
+## - spec for VolumeMount:
+##   https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#volumemount-v1-core
+##
+extraVolumeMounts: []
```

Comment:
Just as a suggestion. If these Volumes and Mounts are not supposed to exist one without another, maybe it's better to add one property to values, and then use it in templates, to make sure that every volume is always mounted?